include/alpaka/kernel/TaskKernelGpuUniformCudaHipRt.hpp

```diff
@@ -364,16 +374,18 @@ namespace alpaka
                           << std::endl;
 #        endif
 
-                auto kernelName = uniform_cuda_hip::detail::
-                    uniformCudaHipKernel<TAcc, TDim, TIdx, TKernelFnObj, remove_restrict_t<std::decay_t<TArgs>>...>;
+                auto kernelName = uniform_cuda_hip::detail::uniformCudaHipKernel<
+                    API,
+                    TAcc,
+                    TDim,
+                    TIdx,
+                    TKernelFnObj,
+                    remove_restrict_t<std::decay_t<TArgs>>...>;
 #        if ALPAKA_DEBUG >= ALPAKA_DEBUG_FULL
-                // hipFuncAttributes not ported from HIP to HIP.
-                // TODO why this is currently not possible
-                //
+#            if 1
                 // Log the function attributes.
-#            if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
```

Comment:
Can we remove the preprocessor directives completely?

Reply:
Yes.